docs/src/main/sphinx/release/release-374.md

```diff
@@ -0,0 +1,81 @@
+# Release 374 (16 Mar 2022)
+
+## General
+
+* Implement parallel read for S3-based spooling in
+  [fault-toleration execution](/admin/fault-tolerant-execution). ({issue}`11050`)
+* Improve performance of queries with `GROUP BY` clauses. ({issue}`11392`)
+* Improve performance of `GROUP BY` with a large number of groups. ({issue}`11011`)
+* Improve handling of queries where individual tasks require lots of memory when
+  `retry-policy` is set to `TASK`. ({issue}`10432`)
+* Produce better query plans by improving cost-based-optimizer estimates in the
+  presence of correlated columns. ({issue}`11324`)
+* Fix memory accounting for queries involving certain variable-width data types
+  such as `varchar` or `varbinary`., Benchmark shows a 0.7%-2.7% CPU efficiency
```

Comment:
Remove extra `,`

Reply:
Also, the mention about benchmarks seems disconnected from the first part, which describes a memory accounting issue. I would just remove that section, and maybe add "Fix memory accounting and improve performance for queries ..." at the front.